x/ccv/consumer/types/genesis_test.go

```diff
@@ -265,17 +282,18 @@ func TestValidateRestartGenesisState(t *testing.T) {
 				nil,
 				nil,
 				types.SlashRequests{},
+				types.LastTransmissionBlockHeight{},
 			},
 			true,
 		},
 		{
 			"invalid restart consumer genesis state: nil initial validator set",
-			types.NewRestartGenesisState("ccvclient", "ccvchannel", nil, nil, nil, nil, params),
+			types.NewRestartGenesisState("ccvclient", "ccvchannel", nil, nil, nil, types.SlashRequests{}, nil, types.LastTransmissionBlockHeight{}, params),
 			true,
 		},
 		{
 			"invalid restart consumer genesis state: invalid params",
-			types.NewRestartGenesisState("ccvclient", "ccvchannel", nil, valUpdates, nil, nil,
+			types.NewRestartGenesisState("ccvclient", "ccvchannel", nil, valUpdates, nil, types.SlashRequests{}, nil, types.LastTransmissionBlockHeight{},
```

Comment:
Are you doing enough testing of the restart genesis state where the handshake hasn't completed?

Reply:
Agree. These genesis validation tests need some adjustments as well.